src/test/java/org/elasticsearch/test/ElasticsearchIntegrationTest.java

```diff
@@ -650,49 +669,12 @@ protected final void afterInternal(boolean afterClass) throws Exception {
         } finally {
             if (!success) {
                 // if we failed here that means that something broke horribly so we should clear all clusters
-                afterTestRule.forceFailure();
+                // TODO: just let the exception happen, WTF is all this horseshit
```

Comment:
do we want to just remove the finally clause?  @javanna do you care to comment here?